README.md

```diff
@@ -12,24 +12,23 @@
 
 ### Features
 
-* **Own your own data** - Don’t ship your data to 3rd party services. Manage your own data and protect your users to maintain your competitive advantage.
-* **Enterprise Grade** - Wasabi has been battle tested in production in fin-tech for 2 years in TurboTax, QuickBooks, Mint.com, etc., powering over 1,000 experiments across Intuit.
-* **High Performance** - Consistent performance response times within 30ms server side.
-* **100% API Driven** - Compatible with any language and environment.
-* **Platform Agnostic** - A uniform, consistent experience across web, mobile, desktop, and also front-end, back-end.
-* **Get up and running in minutes** - Spin up your Wasabi docker in 5 minutes and be in production with the platform, instrumentation, and experiments within a day.
-* **It’s Free** - Don’t pay an arm and a leg for an enterprise-grade platform – the platform is free! 
-* **Real-time assignments** - Assign users into experiments in real-time, to preserve traffic for other parallel A/B tests.
+* **Own your own data** - Wasabi runs on your servers, in the cloud or on-premise, where you have complete control over your data.
+* **Proven** - Wasabi is battle-tested in production at Intuit, Inc., a financial technology company. Wasabi is the experimentation platform for TurboTax, QuickBooks, Mint.com, and other Intuit offerings.
+* **High Performance** - Consistent (P99) server-side response times for user assignments within 30ms.
+* **100% API-Driven** - The Wasabi REST API is compatible with any language and environment.
+* **Platform Agnostic** - Uniform, consistent testing across Web, mobile, desktop, and also front-end, back-end.
+* **Real-time user assignments** - Assign users into experiments in real time, to preserve traffic for other parallel A/B tests.
 * **Cloud and on-premise** - Designed to live in the cloud or in your own data center.
-* **Analytics** - Core analytics functionality and metrics visualization out of the box. 
-* **Administrative UI** - Manage and setup your experiments using our user friendly interface.
+* **Analytics** - Core experiment analytics and metrics visualization out of the box, as well as the ability to send data to your existing analytics infrastructure.
+* **Pluggable** - Well-defined interfaces for plugging in your own access control, sending data to data pipelines, and providing fully custom bucket allocations.
+* **Experiment Management UI** - Setup and manage experiments via a modern Web interface. Management via REST API is also possible.
+* **Dockerized** - Spin up a Wasabi Docker instance in 5 minutes and be in production with the platform, instrumentation, and experiments within a day.
```

Comment:
@toddfast It's more like 15 min with VM docker now. But it's a lot fast (more like 5 min) with native docker on Mac (work in progress) :)